examples/compiler/02_data_regions.ipynb

```diff
@@ -33,7 +31,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "At this point, we have a time-varying 3x3 grid filled with `1's`. Below, we can see the `domain` data values:"
+    "At this point, we have a time-varying 3x3 grid filled with _1's_. Below, we can see the _domain_ data values:"
```

Comment:
I like this change. In practice, now domain/halo/padding are written in italics, while the actual devito objects (such as TimeFunction, Operator, etc) within ` . Correct?

Reply:
exactly

Reply:
nice